sources/logdog/src/log_request.rs

```diff
@@ -44,12 +42,22 @@ const SENSITIVE_SETTINGS_PATTERNS: &[&str] = &[
 /// Returns the list of log requests to run by combining `VARIANT_REQUESTS` and `COMMON_REQUESTS`.
 /// These are read at compile time from files named `logdog.conf` and `logdog.common.conf`
 /// respectively.
-pub(crate) fn log_requests() -> Vec<&'static str> {
-    COMMON_REQUESTS
-        .lines()
-        .chain(VARIANT_REQUESTS.lines())
-        .filter(|&command| !command.is_empty() && !command.trim_start().starts_with('#'))
-        .collect()
+pub(crate) fn log_requests() -> Result<Vec<String>> {
+    Ok(WalkDir::new(REQUESTS_DIR)
+        .into_iter()
+        .collect::<std::result::Result<Vec<_>, _>>()?
+        .into_iter()
+        .filter(|entry| entry.path().is_file())
```

Comment:
Typically for these `.conf.d` directories, the snippets are added in order, e.g.:
```
00-release.conf
10-networkd.conf
20-k8s.conf
```

That doesn't matter a whole lot here, since entries can't really conflict or overwrite each other, but it might still be useful so that all the commands are run in a predictable order. And it 

Also, I would set min and max depth to 1 since we don't expect to recurse into nested dirs.